src/helpers/articles/recordStats.js

```diff
@@ -0,0 +1,20 @@
+import { Statistic } from '../../sequelize/models';
+
+const recordStatistics = async (article) => {
```

Comment:
This line 3 is not fully tested 
<img width="347" alt="Screen Shot 2019-08-30 at 16 59 08" src="https://user-images.githubusercontent.com/50101879/64030884-b4538600-cb47-11e9-9fc0-f74fa394fb78.png">

Reply:
I don't know why it didn't get fully covered but it really does its job of receiving an article as a parameter.